### ISSUE TITLE
x-parser-schema-id getting rendered in html docs

### ISSUE THREAD

**Issue by InTheCloudDan** [asyncapi/html-template#15]
#### Describe the bug
HTML Template example is getting rendered with `x-parser-schema-id: null`. The docs just render with the word `x-parser-schema-id`

#### How to Reproduce
```
streamHeaders:
      Etag:
        description: |
          The RFC7232 ETag header field in a response provides the current entity-
          tag for the selected resource. An entity-tag is an opaque identifier for
          different versions of a resource over time, regardless whether multiple
          versions are valid at the same time. An entity-tag consists of an opaque
          quoted string, possibly prefixed by a weakness indicator.
        example: 411a
      Cache-Control:
        description: The Cache-Control HTTP header holds directives (instructions) for caching in request.
        type: string
        example: no-cache, no-store, must-revalidate
```
under messages I have:

``` 
     headers:
        type: object
        properties:
          $ref: '#/components/schemas/streamHeaders'
```

- Screenshots
![image](https://user-images.githubusercontent.com/10308834/80319603-48611480-87df-11ea-81c8-a1427ed359a9.png)

#### Expected behavior
That `x-parser-schema-id` doesn't render. Could definitely be user error but either way I don't think the internal metadata should be outputted.

**Comment by kurrata** [comment]
I have the same issue.
When using [playground](https://playground.asyncapi.io/?utm_source=editor&utm_medium=editor-topbar&utm_campaign=playground) meta information is added to html output when chaining references it seems
It can be seen in containerCreateResponse while containerCreateRequest  doesn't have it

yaml file
```yaml
asyncapi: 2.0.0
info:
  title: Test
  version: '1.0.0'

          
defaultContentType: application/json

channels:
  container/create:
    description: Create container
    publish:
      message:
        $ref: '#/components/messages/containerCreateRequest'
    subscribe:
      message:
        $ref: '#/components/messages/containerCreateResponse'
        
components:
  schemas:
    Container:
      id:
        type: id
      barcode:
        type: string
        
    BaseRequest:
      action:
        type: string
        description: "Requested action name. Generally it is in <controller>/<action> format"
      uuid:
        type: string
        description: "UUID number used to match request with responses if nececary"
      payload:
        oneOf:
          - type: array
            description: "Request payload"
          - type: None
        
    BaseResponse:
      action:
        type: string
        description: "Requested action name. Generally it is in <controller>/<action> format"
      code:
        type: integer
        description: "Actions response code. It mostly follows http status codes"
        enum:
          - 200
          - 400 
          - 401
          - 403
          - 500
      uuid:
        type: string
        description: "UUID number used to match request with responses if nececary"
      payload:
        oneOf:
          - type: array
            description: "Response payload"
          - type: None
          
  messages:
    containerCreateRequest:
      payload:
        type: object
        required:
        - action
        properties:
          $ref: "#/components/schemas/BaseRequest"
          payload:
            type: object
            properties:
              sender_unit:
                type: string
                description: Unit's id or fcd code
              addressee_unit:
                type: string
                description: Unit's id or fcd code
              consignment_type:
                type: string
                description: containers consignment id or label
              source_id:
                type: integer
                description: System id 

              
            
    containerCreateResponse:
      payload:
        type: object
        required:
        - action
        - code
        properties:
          $ref: "#/components/schemas/BaseResponse"
          payload:
            type: object
            properties:
              $ref: "#/components/schemas/Container"
```

Generated bad section
```json
{
  "payload": {
    "id": null,
    "barcode": "string",
    "x-parser-schema-id": null
  },
  "action": "string",
  "code": 200,
  "uuid": "string"
}
```

**Comment by derberg** [comment]
`x-parser-schema-id` is added by the parser to every schema that has no schema id and it is needed purely for code generation. I totally agree it should not be rendered in docs. Is anyone interested in contributing this functionality? I can help pointing out where it needs to be done.

**Comment by derberg** [comment]
we need to have the proper condition [here](https://github.com/asyncapi/html-template/blob/master/partials/schema-prop.html) to not render any property that starts with `x-parser`

**Comment by derberg** [comment]
video explanation on how to work on this issue https://youtu.be/Iqs_2BiNEEo?t=329

**Comment by derberg** [comment]
@magicmatatjahu yo, I'm not sure if I should reopen this one, how does it look like in react component?

**Comment by magicmatatjahu** [comment]
@derberg Generation for examples are not yet rewritten for the "new" component. I will reopen it and I remember about this issue to check and fix in component :)

**Comment by magicmatatjahu** [comment]
It will be fixed in https://github.com/asyncapi/html-template/pull/166

**Comment by magicmatatjahu** [comment]
Fixed in `@asyncapi/html-template@0.21.0` @InTheCloudDan Reopen if will you found bugs, and sorry for delay 😅
